tests/phpunit/CRM/Financial/Form/PaymentFormsTest.php

```diff
@@ -0,0 +1 @@
+<?php
```

Comment:
@eileenmcnaughton should this have been committed?

Reply:
opps no

Reply:
@seamuslee001 fixed